Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -895,10 +896,6 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
     private func stopAdapter() async throws {
         try await withCheckedThrowingContinuation { (continuation: CheckedContinuation<Void, Error>) in
             adapter.stop { [weak self] error in
-                if let self {
-                    self.handleAdapterStopped()
```

Comment:
This was causing some trouble in the stop pixels not firing, likely because this was causing a delay and the tunnel was killed.

This isn't necessary as it's already done in line 889 anyway.